src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusSerializerExt.cs

```diff
@@ -80,6 +80,10 @@ public static int WriteMetric(byte[] buffer, int cursor, Metric metric, Promethe
                 var tags = metricPoint.Tags;
                 var timestamp = metricPoint.EndTime.ToUnixTimeMilliseconds();
 
+                metricPoint.TryGetExemplars(out var exemplarCollection);
```

Comment:
are you planning to add exemplar support in monotonic sum also in a later PR?

Reply:
Just pushed b412857ce3279780b529d997457987100f840f79 with monotonic sum support